### DISCUSSION TITLE
It'd be great if the app observed track order meta data

### DISCUSSION THREAD

**Discussion by esgeiger** [FossifyOrg/General-Discussion#565] [Ideas]
This is a great app, and I love the feature set, especially the search. One thing that it seems to lack is the ability to play tracks in album order as expressed in file metadata. Right now, it ignores track order metadata and provides no play order settings that I can find.

**Comment by Aga-C** [comment]
Where do you encounter this problem? If an audio file has a track number meta tag, it's respected:
- in Albums tab, it's automatically ordered by the track number
- same in Artists tab
- in other tabs, you can sort by track number (icon with three horizontal bars on the search bar)

**Reply by esgeiger** [reply]
Never mind - the shuffle feature was enabled by default when I installed the app, even after removing and reinstalling. Once I disabled it, playback proceeded in the track order sequence.
